### DISCUSSION TITLE
How to make gif work properly with next/image

### DISCUSSION THREAD

**Discussion by ziyak97** [vercel/next.js#18628] [Help]
If you got to my site https://taskpro-website.vercel.app/ and scroll to the bottom you will see a gif of a lightbulb watering a plant (screenshot attached). This gif is supposed to be a motion gif (gif attached below). It works fine with normal img tags but has no motion when I use the new Image tag. How do i solve this issue?
My next.config.js settings
		images: {
			domains: ['backend.taskproservices.com'],
		},
Any help would be appreciated. Thanks!
![lighbulb](https://user-images.githubusercontent.com/53836911/97809147-3f229a00-1c91-11eb-8fb8-730e772e3151.gif)
<img width="1402" alt="Screenshot 2020-11-01 at 10 25 19 PM" src="https://user-images.githubusercontent.com/53836911/97809149-434eb780-1c91-11eb-85a8-3818245a2d97.png">

**Comment by alex-r89** [comment]
I am also wondering about this! Gifs currently aren't supported afaik :(

**Reply by ziyak97** [reply]
Yeah I have reported the issue hopefully they fix it - https://github.com/vercel/next.js/issues/18683

**Comment by victorbruce** [comment]
> If you got to my site https://taskpro-website.vercel.app/ and scroll to the bottom you will see a gif of a lightbulb watering a plant (screenshot attached). This gif is supposed to be a motion gif (gif attached below). It works fine with normal img tags but has no motion when I use the new Image tag. How do i solve this issue? My next.config.js settings images: { domains: ['backend.taskproservices.com'], }, Any help would be appreciated. Thanks! ![lighbulb](https://user-images.githubusercontent.com/53836911/97809147-3f229a00-1c91-11eb-8fb8-730e772e3151.gif) <img alt="Screenshot 2020-11-01 at 10 25 19 PM" width="1402" src="https://user-images.githubusercontent.com/53836911/97809149-434eb780-1c91-11eb-85a8-3818245a2d97.png">

Were you able to find a solution to this problem, please? Currently facing a similar issue where GIFs run perfectly on localhost but turn s into a still image in production.

**Reply by IlyaKaplan2002** [reply]
Hi, have you found a solution? I have the same problem: on localhost everything is ok, but in production something goes wrong

**Reply by ziyak97** [reply]
It's been awhile but I think I just used a regular img tag for the gif as it seemed like an issue with the Next Image component.

**Reply by IlyaKaplan2002** [reply]
Thank you, I will do the same

**Reply by Shreyas110722** [reply]
use unoptimized={true}
it will work for next image tag only

**Comment by pbredenberg** [comment]
Next/image will try to optimize any image you throw at it by default. It does not support GIFs. The solution for me was to set the `unoptimized` attribute to true on `<Image /> See docs: https://nextjs.org/docs/api-reference/next/image#unoptimized

Example:

```
<Image
  src={'https://domain.tld/image.gif'}
  layout={'responsive'}
  height={175}
  width={175}
  alt={`A cute animal!`}
  unoptimized={true}
/>
```

**Reply by ghost** [reply]
It worked for me. Thankyou soo much
